src/App.js

```diff
@@ -4,9 +4,18 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
 
   render() {
+    const { query } = this.state;
+
+    const movies = moviesFromServer.filter(
+      movie => movie.title.toUpperCase().includes(query.toUpperCase())
+      || movie.description.toUpperCase().includes(query.toUpperCase()),
```

Comment:
usually it's converted to losercase but ok, this works as well

how about converting query at the same time as you write it in the state?